sky/cli.py

```diff
@@ -1685,10 +1687,13 @@ def cost_report(all: bool):  # pylint: disable=redefined-builtin
             nonreserved_cluster_records.append(cluster_record)
 
     status_utils.show_cost_report_table(nonreserved_cluster_records, all)
-
     for cluster_group_name, cluster_record in reserved_clusters.items():
         status_utils.show_cost_report_table(
             [cluster_record], all, reserved_group_name=cluster_group_name)
+
+    click.echo(f'\n{colorama.Fore.WHITE}{colorama.Style.BRIGHT}'
+               f'{total_cost}')
+
```

Comment:
We should also hint that the table is partially showed when the `-a` is not specified.